### PR TITLE
Fix #2268 RTL Support for Navigation Drawer

### LINKED ISSUE #2268: [RTL] High-fi Navigation Drawer
`Navigation Drawer` needs RTL support as shown in below screen.
HP (Side Menu) – RTL

Make sure that this works correctly in mobile (landscape + portrait) and tablet (landscape + portrait).

## References
Mobile Portrait: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/grid/
Mobile Landscape: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/grid/
Tablet: https://xd.adobe.com/view/d405de00-a871-4f0f-73a0-f8acef30349b-a234/grid/
Wiki: https://github.com/oppia/oppia-android/wiki/Working-on-UI

### PR DESCRIPTION
**Pull request by imraghav20**
## Explanation

Fixes #2268 

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

Please find the concerned screenshots related to the PR

In portrait mode
WhatsApp Image 2021-04-06 at 9 20 21 AM
<br>
In landscape mode
WhatsApp Image 2021-04-06 at 9 19 57 AM
WhatsApp Image 2021-04-06 at 9 19 57 AM (1)

### CODE REVIEW

File: app/src/main/res/layout/nav_header_navigation_drawer.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
       android:fontFamily="sans-serif"
       android:text="@{@plurals/completed_story_count(viewModel.completedStoryCount, viewModel.completedStoryCount).concat(@string/bar_separator).concat(@plurals/ongoing_topic_count(viewModel.ongoingTopicCount, viewModel.ongoingTopicCount))}"
       android:textColor="@color/white"
-      android:textSize="14sp" />
+      android:textSize="14sp"
+      android:textAlignment="viewStart"/>
```
Comment (FareesHussain) [inline app/src/main/res/layout/nav_header_navigation_drawer.xml]:
```suggestion
      android:textAlignment="viewStart" />
```

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@imraghav20 NIT fix 
rest, LGTM

File: app/src/main/res/layout/nav_header_navigation_drawer.xml (hunk 1 above)
Comment (imraghav20) [reply app/src/main/res/layout/nav_header_navigation_drawer.xml]: Committed changes

Comment (FareesHussain) [reply app/src/main/res/layout/nav_header_navigation_drawer.xml]: You can resolve this

File: app/src/main/res/layout/nav_header_navigation_drawer.xml [hunk 2]
```diff
@@ -51,6 +51,7 @@
       android:fontFamily="sans-serif"
       android:text="@{@plurals/completed_story_count(viewModel.completedStoryCount, viewModel.completedStoryCount).concat(@string/bar_separator).concat(@plurals/ongoing_topic_count(viewModel.ongoingTopicCount, viewModel.ongoingTopicCount))}"
       android:textColor="@color/white"
-      android:textSize="14sp" />
+      android:textSize="14sp"
+      android:textAlignment="viewStart" />
```
Comment (rt4914) [inline app/src/main/res/layout/nav_header_navigation_drawer.xml:55]: You can remove this change. It is not needed.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@imraghav20 Please add screenshots to your description.

For creating RTL screenshot you can follow these guidelines:
https://github.com/oppia/oppia-android/wiki/RTL-Guidelines

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@imraghav20 change the PR description according to the checklist

**Comment by imraghav20** [comment]
> @imraghav20 Please add screenshots to your description.
> 
> For creating RTL screenshot you can follow these guidelines:
> https://github.com/oppia/oppia-android/wiki/RTL-Guidelines

@rt4914 I have already added the screenshots in description.

**Comment by imraghav20** [comment]
> @imraghav20 change the PR description according to the checklist

Yes @FareesHussain made the changes

File: app/src/main/res/layout/nav_header_navigation_drawer.xml (hunk 2 above)
Comment (imraghav20) [reply app/src/main/res/layout/nav_header_navigation_drawer.xml:55]: On removing this change, the text shifts towards left in RTL view.

Comment (FareesHussain) [reply app/src/main/res/layout/nav_header_navigation_drawer.xml:55]: Just to confirm - 
you are changing pseudo locale language to check the changes right

**Review by FareesHussain — APPROVED** [review]
Rest LGTM, Thanks

File: app/src/main/res/layout/nav_header_navigation_drawer.xml (hunk 2 above)
Comment (imraghav20) [reply app/src/main/res/layout/nav_header_navigation_drawer.xml:55]: Sorry @FareesHussain I did not get you

Comment (rt4914) [reply app/src/main/res/layout/nav_header_navigation_drawer.xml:55]: @imraghav20 Check [this wiki](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines) to enable pseudo locale which helps to check RTL UI. Also, update screenshots by using the PseudoLocale.

**Comment by anandwana001** [comment]
@imraghav20 any update on this?

**Comment by anandwana001** [comment]
Closing this pr as no further update.

**Pull request closed without merging**